pages/common/composer-audit.md

```diff
@@ -0,0 +1,25 @@
+# composer audit
+
+> Analyze a PHP project's dependencies to detect known security vulnerabilities and lists affected packages.
+> See also: `composer`.
+> More information: <https://getcomposer.org/doc/03-cli.md#audit/>.
+
+- Check for security vulnerabilities in your current project:
+
+`composer audit`
+
+- Omit dev dependencies in the audit:
+
+`composer audit --no-dev`
+
+- Filter vulnerabilities by output format:
+
+`composer audit --format={{table|plain|json|summary}}`
+
+- Output audit results to a file in JSON format:
+
+`composer audit --format=json > audit_report.json`
+
+- Verify whether a specific package in your project is affected by security issues:
+
+`composer audit {{vendor/package}}`
```

Comment:
What is the format here? If the vendor and package must be separated with a slash, I recommend putting them in their own placeholders.
```suggestion
`composer audit {{vendor}}/{{package}}`
```